frontend/packages/knative-plugin/integration-tests/tests/serverless.scenario.ts

```diff
@@ -1,36 +1,25 @@
-import { execSync } from 'child_process';
-import { protractor, by, browser, ExpectedConditions as until } from 'protractor';
+import { browser, ExpectedConditions as until } from 'protractor';
 import {
-  appHost,
-  checkLogs,
-  checkErrors,
-  testName,
-  retry,
-} from '../../../../integration-tests/protractor.conf';
+  switchPerspective,
+  Perspective,
+  sideHeader,
+} from '../../../dev-console/integration-tests/views/dev-perspective.view';
```

Comment:
Why are you using relative imports here? Why not `@console/dev-console/integration-tests/views/dev-perspective.view`

Reply:
This is the same way we have been using. We never used that type of import that you have suggested anywhere except for the protractor.conf 
Is it important? How does it affect?